### PR TITLE
Brazil

### PR DESCRIPTION
**Pull request by ofrancon**
Added population estimates for states of Brazil
Also re-generated the `covid_xprize/validation/data/2020-09-30_historical_ip.csv` validation file to include states of Brazil:

### CODE REVIEW

File: covid_xprize/examples/predictors/lstm/data/brazil_populations.csv [hunk 1]
```diff
@@ -0,0 +1,28 @@
+GeoID,Population
```
Comment (dsargent) [inline covid_xprize/examples/predictors/lstm/data/brazil_populations.csv:1]: Is there a way to indicate the source of the data? Pandas is okay with comments in CSV files, right?

Comment (ofrancon) [reply covid_xprize/examples/predictors/lstm/data/brazil_populations.csv:1]: I don't think CSV files have a standard way of specifying comments...

**Review by dsargent — APPROVED** [review]

File: covid_xprize/examples/predictors/lstm/data/brazil_populations.csv (hunk 1 above)
Comment (EKMeyerson) [reply covid_xprize/examples/predictors/lstm/data/brazil_populations.csv:1]: Maybe a little readme in the this `data` dir?

**Review by EKMeyerson — COMMENTED** [review]
lgtm; with the note that including the brazil regions breaks other predictors that rely on older copies of the oxford data.

**Pull request merged**